src/Codeception/Module/Doctrine2.php

```diff
@@ -424,7 +424,7 @@ protected function buildAssociationQuery($qb, $assoc, $alias, $params)
             if ($val === null) {
                 $qb->andWhere("s.$key IS NULL");
             } else {
-                $paramname = "s__$key";
+                $paramname = str_replace(".", "", "s_$key");
```

Comment:
I didn't work with Doctrine for a while but why `s__` is changed to `s_`. Is that ok?